test/test_ngraph_exec.cpp

```diff
@@ -36,6 +41,14 @@ namespace tensorflow {
 namespace ngraph_bridge {
 namespace testing {
 
+string THISDIR =
+#if defined(WIN32) || defined(_WIN32) || \
+    defined(__WIN32) && !defined(__CYGWIN__)
+    std::regex_replace(__FILE__, std::regex("^(.*)\\[^\\]+$"), "$1\\");
+#else
+    std::regex_replace(__FILE__, std::regex("^(.*)/[^/]+$"), "$1/");
+#endif
+
```

Comment:
Why is this required?

Reply:
I removed path dependency of running gtest from some other folder e.g. source root. Now, it will always find the associated pbtxt files in the same dir as the test script.

Reply:
This creates a dependency on the source. If you want to not use a path relative to CWD, please use a path relative to the test executable.

Reply:
I will revert to older approach of copying pbtxt files to build output.